app/views/cases/_case_request.html.slim

```diff
@@ -10,6 +10,9 @@
     - if case_details.message.present?
       p.message-body
         = show_hide_message(case_details)
+    - if !case_details.message.present? and case_details.offender_sar?
+      p.messag-body 
```

Comment:
Small typo

```suggestion
      p.message-body 
```

The number of conditionals is becoming too much in this file, I think. At this point I'd think about creating an offender_sar flavoured partial and using that instead

Reply:
Oh, also is the `and` intentional here? There's a difference between using it for logical operations versus flow control. If it's the former, and I think it might be here, it's preferred to use `&&`

This contrasts with situations like `render json: @user and return` etc

Reply:
```if !case_details.message.present? and case_details.offender_sar?```

is the `and` intentional here? There's a difference between using it for logical operations versus flow control. If it's the former, and I think it might be here, it's preferred to use `&&`

This contrasts with situations like `render json: @user and return` etc

Also, we'd use if if `case_details.message.empty?` in place of `!case_details.message.present?`

Reply:
Just thinking about it, maybe we should add a view spec for the logic displaying the alternate message.